server/graphql/invoice/src/mutations/outbound_shipment/update.rs

```diff
@@ -176,7 +174,6 @@ fn map_error(error: ServiceError) -> Result<UpdateErrorInterface> {
         ServiceError::OtherPartyDoesNotExist => BadUserInput(formatted_error),
         ServiceError::DatabaseError(_) => InternalError(formatted_error),
         ServiceError::InvoiceLineHasNoStockLine(_) => InternalError(formatted_error),
-        ServiceError::UpdatedInvoicenDoesNotExist => InternalError(formatted_error),
```

Comment:
Ohh, shoot, once again I was misleading, this is a different error (i'll be more attentive next time). Sorry can you reinstate (this one should be an internal error in case for some reason the insertion didn't work

Reply:
Sorry for misunderstanding ;-;. Added back in here [5bec839](https://github.com/openmsupply/open-msupply/pull/329/commits/5bec8391d532578b1ad436ff67692fa7a1e6140e)